src/controller/data_model/controller-clusters.zap

```diff
@@ -16861,7 +17025,7 @@
               "side": "server",
               "type": "LineupInfoStruct",
               "included": 1,
-              "storageOption": "RAM",
+              "storageOption": "External",
```

Comment:
these changes from RAM to External seem unexpected. At the same time I imagine controller-clusters does not care about them. Would feel safer if these deltas would not exist though.

Reply:
These changes are expected if there have been changes to some metadata about which things are forced to be AttributeAccessInterface and then no one ever updated this file (by running ZAP UI or running the update-zap-file script).

Reply:
So I assume this change is okay then and the discussion could be marked as resolved @bzbarsky-apple?